src/js/lib/sheetNameFromShortId.js

```diff
@@ -1,5 +1,5 @@
 export default function sheetNameFromShortId(docsArray, shortUrl) {
     const map = new Map(docsArray);
     const docName = map.get(shortUrl);
-    return docName ? docName : 'gun-nation';
+    return docName ? docName : 'internet-warriors';
```

Comment:
This is a workaround to the fact that now we are relying on `pageId` we don't know this until the day of publication, so we default to the latest doc.